intellij-plugin-structure/structure-base/src/main/kotlin/com/jetbrains/plugin/structure/base/problems/ProblemSolutionHint.kt

```diff
@@ -0,0 +1,5 @@
+package com.jetbrains.plugin.structure.base.problems
+
+import java.net.URL
+
+data class ProblemSolutionHint(val example: String?, val documentationUrl: URL?)
```

Comment:
move to com.jetbrains.plugin.structure.base.plugin.PluginProblem? not sure about `URL`

Reply:
Moved to the `PluginProblem` file. It will be a separate class for now.

Re URL: This is a very first effort to improve messages on errors and warnings. It will be improved on further PRs and integrated with the rest of the ecosystem.

Reply:
I meant using `URL` instead of `String` to store the link